mlrun/api/api/endpoints/healthz.py

```diff
@@ -46,3 +48,25 @@ def _get_config_value_if_not_default(config_key):
         return None
     else:
         return current_config_value
+
+
+cached_nuclio_version = None
```

Comment:
@Hedingber  @omesser 
that means, when patching nuclio, mlrun needs to be restarted

Reply:
Yup